### ISSUE TITLE
Why do you skip normalization on first block of Generator on wgan.py?

### ISSUE THREAD

**Issue by MLSlayer** [eriklindernoren/PyTorch-GAN#68]
I been seeing this skipping happening elsewhere in other's code. 

https://github.com/eriklindernoren/PyTorch-GAN/blob/master/implementations/wgan/wgan.py#L51

Is there a reason you toggle BatchNorm off here?

**Comment by bsridatta** [comment]
The first layer of the generator and the output layer of the discriminator/critic should not have BN. Cant remember the source

**Comment by MLSlayer** [comment]
got it, thanks!
